packages/react/src/hooks/useAmplify.ts

```diff
@@ -1,17 +0,0 @@
-import * as React from 'react';
```

Comment:
Note: I don't believe this file is used in Studio, but it's not used by us, so I'm cleaning it up. All the other files I believe are Studio related.